src/vs/workbench/contrib/files/browser/editors/fileEditorTracker.ts

```diff
@@ -244,7 +243,7 @@ export class FileEditorTracker extends Disposable implements IWorkbenchContribut
 		});
 	}
 
-	private getIndexOfPath(path: string, candidate: string): number {
+	private getIndexOfPath(path: string, candidate: string, ignoreCase: boolean): number {
```

Comment:
@aeschli I would just remove this parameter `ignoreCase: boolean` and inline it to `resources.hasToIgnoreCase(resource)`?